### PR TITLE
Use tests/ci/style_check.py to check the style

### PR DESCRIPTION
**Pull request by Felixoid**
### Changelog category (leave one):
- Documentation (changelog entry is not required)

### Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):
A follow-up for #64493

### Documentation entry for user-facing changes

- [ ] Documentation is written (mandatory for new features)

> Information about CI checks: https://clickhouse.com/docs/en/development/continuous-integration/

#### CI Settings (Only check the boxes if you know what you are doing):
- [ ]  Allow: All Required Checks
- [ ]  Allow: Stateless tests
- [ ]  Allow: Stateful tests
- [ ]  Allow: Integration Tests
- [ ]  Allow: Performance tests
- [ ]  Allow: All NOT Required Checks
- [ ]  Allow: batch 1, 2 for multi-batch jobs
- [ ]  Allow: batch 3, 4, 5, 6 for multi-batch jobs
---
- [ ]  Exclude: Style check
- [ ]  Exclude: Fast test
- [ ]  Exclude: Integration Tests
- [ ]  Exclude: Stateless tests
- [ ]  Exclude: Stateful tests
- [ ]  Exclude: Performance tests
- [ ]  Exclude: All with ASAN
- [ ]  Exclude: All with Aarch64
- [ ]  Exclude: All with TSAN, MSAN, UBSAN, Coverage
---
- [ ]  Do not test
- [ ]  Upload binaries for special builds
- [ ]  Disable merge-commit
- [ ]  Disable CI cache

### CODE REVIEW

**Comment by robot-clickhouse-ci-1** [comment]
*This is an automated comment for commit 0fe2c1dae4ed8dffbe3e877344c17dc66dad3dbd with description of existing statuses. It's updated for the latest CI running*

[⏳ Click here](https://s3.amazonaws.com/clickhouse-test-reports/64808/0fe2c1dae4ed8dffbe3e877344c17dc66dad3dbd/ci_running.html) to open a full report in a separate page

<table>
<thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody>
<tr><td>A Sync</td><td>If it fails, ask a maintainer for help</td><td>⏳ pending</td></tr>
<tbody>
</table>
<details><summary>Successful checks</summary>
<table>
<thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody>
<tr><td>PR Check</td><td>Checks correctness of the PR's body</td><td>✅ success</td></tr>
<tbody>
</table>
</details>

File: docs/en/development/continuous-integration.md [hunk 1]
```diff
@@ -71,7 +71,7 @@ If it fails, fix the style errors following the [code style guide](style.md).
 ```sh
 mkdir -p /tmp/test_output
 # running all checks
-docker run --rm --volume=.:/ClickHouse --volume=/tmp/test_output:/test_output -u $(id -u ${USER}):$(id -g ${USER}) --cap-add=SYS_PTRACE clickhouse/style-test
+python3 tests/ci/style_check.py --no-push
 
 # run specified check script (e.g.: ./check-mypy)
```
Comment (maxknv) [inline docs/en/development/continuous-integration.md:76]: could be removed till the end of Style check I think

Comment (Felixoid) [reply docs/en/development/continuous-integration.md:76]: Sorry, I don't get it. It's the doc for https://clickhouse.com/docs/en/development/continuous-integration#running-style-check-locally

What could be removed?

**Review by alexey-milovidov — APPROVED** [review]

File: docs/en/development/continuous-integration.md (hunk 1 above)
Comment (maxknv) [reply docs/en/development/continuous-integration.md:76]: from `# run specified check script (e.g.: ./check-mypy)` and till the end of style check. too detailed.
never mind, maybe next time...

Comment (Felixoid) [reply docs/en/development/continuous-integration.md:76]: Unless we add arguments to run them from `style_check.py,` it's useful info for external devs. After that yes, the docker workarounds can go

**Pull request merged**